src/Microsoft.AspNet.Http.Abstractions/HttpContextAccessor.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+#if DNX451
+using System.Runtime.Remoting.Messaging;
+using System.Runtime.Remoting;
+#elif DNXCORE50
+using System.Threading;
+#endif
+
+namespace Microsoft.AspNet.Http
```

Comment:
Sounds reasonable, but there's nothing else in Internal in HttpAbstractions currently, but I can create the folder and move it there if you think its the right thing do